ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -84,6 +86,35 @@ da_scala_binary(
     ],
 )
 
+da_scala_binary(
+    name = "ledger-acceptance",
+    srcs = glob([
+        "src/main/scala/com/daml/ledger/acceptance/**/*.scala",
+    ]),
+    data = [
+        "//ledger/ledger-api-integration-tests:SemanticTests.dar",
+        "//ledger/sandbox:Test.dar",
+    ],
+    main_class = "com.daml.ledger.acceptance.Main",
+    resources = [
+        "src/main/resources/logback.xml",
+    ],
+    tags = [
+        "maven_coordinates=com.daml.ledger.testtool:ledger-api-test-tool:__VERSION__",
```

Comment:
Did you want to use the same maven coordinates because the new tool will eventually act as a drop-in replacement for the existing test tool?

Reply:
No, it's a whole lotta copy :spaghetti:.

Reply:
I'm adopting everything from the test tool, so the answer to your question turned to a yes.